lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -42,6 +42,7 @@ def initialize(file)
         raise KafoParsers::ParseError, "'#{command}' returned error\n#{@raw_json}"
       end
 
+      Encoding.default_external = Encoding::UTF_8 if defined?(Encoding) && Encoding.respond_to?(:default_external=)
```

Comment:
Yeah, overly defensive - I left it in by accident in c04662f when fixing it for 1.8.7, let me remove it from both places.